source/extensions/filters/common/local_ratelimit/local_ratelimit_impl.cc

```diff
@@ -123,17 +128,61 @@ bool LocalRateLimiterImpl::requestAllowedHelper(const TokenState& tokens) const
   return true;
 }
 
-bool LocalRateLimiterImpl::requestAllowed(
+StatusOr<std::reference_wrapper<const LocalRateLimiterImpl::LocalDescriptorImpl>>
```

Comment:
I think perhaps using `OptRef` might be a better type for this since we never really read the status message?

Reply:
Thanks, I will change it.